docs/docusaurus/docs/cloud/deploy_gx_agent.md

```diff
@@ -179,7 +179,7 @@ You can deploy the GX Agent in any environment in which you create Kubernetes cl
     spec:
       containers:
        name: gx-agent
-        image: greatexpectations/agent:latest
+        image: greatexpectations/agent:stable
```

Comment:
the docker command should also be updated, see https://github.com/great-expectations/great_expectations/pull/9783/files#diff-698bcd3d5c27523a0e755f06812fd67e394992e4b453a838418a30982d2bf64dR126

Reply:
Thanks for the catch :) updated